addon/validator/custom-formats/utils.js

```diff
@@ -0,0 +1,36 @@
+export const networkMaskMax = 32
+export const networkMaskMin = 0
+
+/**
+ * Convert decimal value to binary representation
+ * @param {Number} decimal - decimal value to convert to binary
+ * @returns {String} string containing binary representation
+ */
+export function decimalToBinary (decimal) {
+  return (decimal >>> 0).toString(2)
```

Comment:
Might be a good idea comment why the shift is necessary.  It wasn't obvious at first.